### PR TITLE
Release 0.1.0 preparation

### PR DESCRIPTION
**Pull request by squaremo**
- start the changelog up
 - bump the gotk packages* and APIs

*but not source-controller, yet; 0.5.1 does not compile straight-forwardly, and I don't need the bits that got updated, updated.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Changelog
+
+## 0.1.0
+
+**Release date:** 2020-12-10
+
+This is the first ever release of image-automation-controller and its
```
Comment (stefanprodan) [inline CHANGELOG.md:7]:
```suggestion
This is the first prerelease of image-automation-controller and its
```

Comment (squaremo) [reply CHANGELOG.md:7]: It's not a prerelease, unless you want to go through 0.1.0-rc* first

Comment (stefanprodan) [reply CHANGELOG.md:7]: We mark all 0.x versions as prereleases in GitHub https://github.com/fluxcd/source-controller/releases

Comment (squaremo) [reply CHANGELOG.md:7]: Ah, everything is a prerelease at present (until 1.0?). OK, got it.

File: go.mod [hunk 2]
```diff
@@ -7,12 +7,12 @@ replace github.com/fluxcd/image-automation-controller/api => ./api
...
 	github.com/fluxcd/image-reflector-controller v0.0.0-20200810165546-c2265d9b49b9
-	github.com/fluxcd/pkg/apis/meta v0.4.0
+	github.com/fluxcd/pkg/apis/meta v0.5.0
 	github.com/fluxcd/pkg/gittestserver v0.0.2
-	github.com/fluxcd/pkg/runtime v0.3.1
+	github.com/fluxcd/pkg/runtime v0.4.0
 	github.com/fluxcd/source-controller v0.3.0
 	// If you bump this, change TOOLKIT_VERSION in the Makefile to match
-	github.com/fluxcd/source-controller/api v0.3.0
+	github.com/fluxcd/source-controller/api v0.5.0
```
Comment (hiddeco) [inline go.mod:15]: This can be `v0.5.1`, saves us another line in the `go.sum` of `fluxcd/flux`.

**Review by hiddeco — APPROVED** [review]

File: go.mod (hunk 2 above)
Comment (squaremo) [reply go.mod:15]: It imports both, but I don't want to upgrade source-controller/pkg past the libgit2 change yet.

**Pull request merged**